### DISCUSSION TITLE
The build log for Codespaces never updates with data

### DISCUSSION THREAD

**Discussion by krainboltgreene** [community/community#4225] [Codespaces] [answered]
I fully customized my `.devcontainer` and while building the log portion of the UI never actually outputs anything.

<img width="1083" alt="Screen Shot 2021-06-21 at 1 45 02 AM" src="https://user-images.githubusercontent.com/334809/122733784-5ae3ac00-d232-11eb-9160-fc1c9a38cb50.png">

This is half an hour into the build.

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @krainboltgreene does a browser refresh fix this for you (it has worked for me in the past)? Are you still experiencing this?

**Reply by krainboltgreene** [reply]
No, sadly, but it does seem intermittent. Sometimes I get output sometimes I don't.

**Reply by tanmayeekamath** [reply]
Thanks for confirming! The next time this happens, can you please file a support ticket by going to [GitHub Support.](https://support.github.com/)
Our team can help investigate the underlying issue by taking a look at your specific instance :)
